src/Contract/Operation/Matchingable.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/**
+ * For the full copyright and license information, please view
+ * the LICENSE file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace loophp\collection\Contract\Operation;
+
+use Doctrine\Common\Collections\Criteria;
+use Doctrine\Common\Collections\Selectable as DoctrineCollectionsSelectable;
```

Comment:
```suggestion
use Doctrine\Common\Collections\Criteria;
use Doctrine\Common\Collections\Selectable;
```

I don't see why the alias is needed if we don't have an alias for the other import